templates/components/multiagent/python/app/examples/workflow.py

```diff
@@ -171,7 +180,10 @@ async def write(
         if ev.is_good or too_many_attempts:
             # too many attempts or the blog post is good - stream final response if requested
             result = await self.run_agent(
-                ctx, writer, ev.input, streaming=ctx.data["streaming"]
+                ctx,
+                writer,
+                f"Trim the review from the reviewer and return only the blog post content. Here is the content: {ev.input}",
+                streaming=ctx.data["streaming"],
```

Comment:
don't we store the generated content in `ctx.data["result"]`?
```suggestion
                f"Return only the blog post content. Here is the content: {ctx.data["result"]}",
                streaming=ctx.data["streaming"],
```

Reply:
It's easier to follow the workflow with `ctx.data["result"]`, but using `ev.input` is a good choice too since it includes the reviewer's suggestions for improving the post, making the final response even better.

Reply:
fair point as this is the writer - but then the prompt should be different (not just removing the review but applying the reviewers suggestions)

Reply:
>not just removing the review but applying the reviewers suggestions 

Good idea